integrations/intercom.mdx

```diff
@@ -49,3 +49,8 @@ When ingesting conversations, we collect the following data and attributes from
 - Message text content (only those of type `Comment` that are **not redacted**)
 - Message Authors (Name, Email, Type)
 
+### Pricing Volume Impact
```

Comment:
My one thought is for these tables, to have a link back to the pricing/volume-calculation page. That way if someone lands on this page (and similar integration pages) and sees this, they have some place to go to understand more of what this means!